ApplicationPattern.yaml

```diff
@@ -1308,6 +1397,7 @@ paths:
                 oam-log-application: 'OamLog'
                 oam-log-application-release-number: '2.0.0'
                 oam-log-operation: '/v1/record-oam-request'
+                oam-log-protocol: 'HTTP'
```

Comment:
line 1452: callback uri definition for OamRequestCausesLoggingRequest has protocol https which needs to be updated to reference oam-log-protocol {$request.body#oam-log-protocol}://...

Reply:
Changed as proposed. Many thanks for the concrete proposal.